db/upgrade-X.X.X-X.Y.Z.sql

```diff
@@ -1,3 +1,33 @@
 --
 -- PacketFence SQL schema upgrade from X.X.X to X.Y.Z
 --
+
+--
+-- Add a primary key to the radacct_log
+--
+
+ALTER TABLE `radacct_log` ADD COLUMN `radacct_log_id` int NOT NULL PRIMARY KEY AUTO_INCREMENT FIRST;
```

Comment:
Sort of agree with @julsemaan here.
There is no standard, but everyone seems to use just table.id.